### PR TITLE
Check WHODAS

### PR DESCRIPTION
**Pull request by ibevers**
- [x] create two activities. one that is the 36 item scale and one that then subsets it to 12 items. 

the way reproschema works is that an activity can reuse items from a different place.

"as we do this work we should only have one set of items, not copies of it. that also will tell us that the 12 items items in this case are a subset of the 36."

### CODE REVIEW

**Comment by ibevers** [comment]
@satra 
Which WHODAS?

[WHODAS 2.0 (World Health Organization Disability Schedule 2.0, 36-item version, self-administered)](https://www.psychiatry.org/getmedia/5f4794de-8d66-4434-99f3-784a3fd1e567/APA-DSM5TR-WHODAS2SelfAdministered.pdf) (also available in print book)

[WHODAS 2.0 (World Health Organization Disability Schedule 2.0, 36-item version, proxy-administered)](https://www.psychiatry.org/getmedia/f1e7a9ff-431f-48ff-9284-9be7a2809c4e/APA-DSM5TR-WHODAS2ProxyAdministered.pdf)

**Comment by ibevers** [comment]
From conversation with Satra during lab meeting:
- 36 item WHODAS should refer to items in the 12 item version
- create different activities for the self and proxy administered versions
- Don't duplicate data

**Comment by ibevers** [comment]
Some WHODAS self-administered items differ from proxy-admin items. Example:

Screenshot 2024-04-17 at 10 43 41 AM

Screenshot 2024-04-17 at 10 43 50 AM

Source: https://www.psychiatry.org/getmedia/5f4794de-8d66-4434-99f3-784a3fd1e567/APA-DSM5TR-WHODAS2SelfAdministered.pdf

**Comment by ibevers** [comment]
- [ ] Copy 12 item version to start proxy and self versions
 - [ ] Update general info for each
 - [ ] Add items for each

**Comment by satra** [comment]
> the 12-item WHODAS is only provided by NDA. Since we want the administered tests to be the APA versions, and we will create SSSOM mappings, should I delete the 12-item WHODAS under the assumption that we can have a mapping to create it from the 36-item WHODASs later?

it's also provided by WHO: https://www.cms.gov/files/document/whodas-20-instrument.pdf

if you want to move, perhaps do a `git move` to move the items from the current whodas-12 into whodas-36. this will keep the git history of those items intact, which is important as a principle for the reproschema library.

**Comment by ibevers** [comment]
- [ ] move whodas-12 items into whodas-36

**Comment by ibevers** [comment]
Undoing the git move because WHODAS12 items are not actually a subset of WHODAS36:

WHODAS12:
Screenshot 2024-04-30 at 11 57 42 AM
Source (page 57): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1

WHODAS36:
APA:
Screenshot 2024-04-30 at 11 58 49 AM  

Source: https://www.psychiatry.org/getmedia/5f4794de-8d66-4434-99f3-784a3fd1e567/APA-DSM5TR-WHODAS2SelfAdministered.pdf

WHO:
Screenshot 2024-04-30 at 12 00 53 PM  

Source (page 54): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1

**Comment by ibevers** [comment]
Screenshot 2024-05-09 at 6 52 38 PM
New validation passes^^

File: activities/WHODAS36_P/items/WHODAS36_P_01 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc4/contexts/generic",
+    "@type": "reproschema:Field",
+    "@id": "WHODAS36_P_01",
+    "prefLabel": "WHODAS 2.0 Proxy Q1",
+    "description": "Question 1 World Health Organization Disability Assessment Schedule (WHODAS 2.0) Proxy-Administered Version",
+    "schemaVersion": "1.0.0-rc4",
+    "version": "0.0.1",
+    "question": "Concentrating on doing something for ten minutes?",
```
Comment (satra) [inline activities/WHODAS36_P/items/WHODAS36_P_01:9]: isn't this exactly the same as the self-administered question. if so one of them should be removed. what's important for mapping items is the question and responses. if those are same, make the questions the same. the description can be adjusted to reflect that.

**Comment by satra** [comment]
also where is the WHODAS-12? shouldn't it be mapping to these items? and is the one used for nimh-minimal?

File: activities/WHODAS36_P/items/WHODAS36_P_01 (hunk 1 above)
Comment (ibevers) [reply activities/WHODAS36_P/items/WHODAS36_P_01:9]: I only checked if the questions in one are a subset of the other. I did not delete questions in the intersection in one and update the schema references and description accordingly. I will do that. @satra

**Comment by ibevers** [comment]
I manually checked WHODAS-12 against the pdf and didn't see any discrepancies. Consistent with the approach I took for the self and proxy 36 question versions, I only checked if the questions in 12 are a subset of 36 self. I will delete items in the intersection and update the schema. @satra 

> Undoing the git move because WHODAS12 items are not actually a subset of WHODAS36:
> 
> WHODAS12: Screenshot 2024-04-30 at 11 57 42 AM Source (page 57): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1
> 
> WHODAS36: APA: Screenshot 2024-04-30 at 11 58 49 AM
> 
> Source: https://www.psychiatry.org/getmedia/5f4794de-8d66-4434-99f3-784a3fd1e567/APA-DSM5TR-WHODAS2SelfAdministered.pdf
> 
> WHO: Screenshot 2024-04-30 at 12 00 53 PM
> 
> Source (page 54): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1

**Comment by ibevers** [comment]
Screenshot 2024-05-10 at 2 40 23 PM

File: activities/WHODAS36_P/items/WHODAS36_P_01 (hunk 1 above)
Comment (ibevers) [reply activities/WHODAS36_P/items/WHODAS36_P_01]: @satra fixed

**Comment by ibevers** [comment]
> I manually checked WHODAS-12 against the pdf and didn't see any discrepancies. Consistent with the approach I took for the self and proxy 36 question versions, I only checked if the questions in 12 are a subset of 36 self. I will delete items in the intersection and update the schema. @satra
> 
> > Undoing the git move because WHODAS12 items are not actually a subset of WHODAS36:
> > WHODAS12: Screenshot 2024-04-30 at 11 57 42 AM Source (page 57): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1
> > WHODAS36: APA: Screenshot 2024-04-30 at 11 58 49 AM
> > Source: https://www.psychiatry.org/getmedia/5f4794de-8d66-4434-99f3-784a3fd1e567/APA-DSM5TR-WHODAS2SelfAdministered.pdf
> > WHO: Screenshot 2024-04-30 at 12 00 53 PM
> > Source (page 54): https://iris.who.int/bitstream/handle/10665/43974/9789241547598_eng.pdf?sequence=1

@satra done

**Pull request merged**